File: TrackProyectosWebAPI/DTOs/EstadisticaDTO.cs
using System.Collections.Generic;

namespace TrackProyectosWebAPI.DTOs
{
    public class EstadisticaDTO
    {
        public int CantidadProyectos  { get; set; }

        public int CantidadProyectosTermiandos { get; set; }
        public int HorasMes { get; set; }

        public int HorasSemana { get; set; }

        public int HorasTotales { get; set; }

        //Lista con las horasDTO de la ultima semana
        public IList<HoraDTO> HorasDTOSemana { get; set; }

        //Lista de 7 enteros donde cada uno representa la cantidad de horas trabajadas durante 
        //un dia de la ultima semana
        public IList<int> HorasDiarias { get; set; }

    }
}

File: TrackProyectosFront/src/app/_models/EstadisticaDTO.ts
import { HoraDTO } from './HoraDTO';

export class EstadisticaDTO{

    cantidadProyectos: number;
    cantidadProyectosTermiandos: number;
    horasMes: number;
    horasTotales: number;
    horasSemana: number;
    horasDTOSemana: Array<HoraDTO>;
    horasDiarias: Array<number>;
    
}